src/Devices/Notebooks.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace Devices;
+
+class Notebooks extends Device {
```

Comment:
Why "Notebooks" (in plural)? Does this class contains data about many notebooks? 
Maybe it's worth to rename to "Notebook"?